bike_rack/parse_utils.py

```diff
@@ -10,14 +38,29 @@ def __init__(self, pdf_path):
         self.page_count = self.reader.getNumPages()
         self.date = self.parse_date(pdf_path)
         self.meeting_date = self.date.strftime("%Y-%m-%d")
+        self.parsed_text = None
 
 
     def read_pdf(self, pdf_path):
         file = open(pdf_path, "rb")
         reader = PdfFileReader(file, strict=False)
         return reader
 
+
     def parse_date(self, pdf_path):
         date_str = pdf_path.stem
         date = dt.datetime.strptime(date_str, "%Y_%m_%d")
         return date
+
+
+    def parse_pages(self, range_type):
+        text = ""
+        for page in self.reader.pages:
+            text += page.extractText().strip()
```

Comment:
I lean toward keeping parsing and cleaning separate in two methods: 

- `Minutes.parse_pages()` just loops through the pages and extracts, then returns attribute `raw_text` (str)
- `Minutes.clean_pages()` replaced the bad chars, removes doubled white spaces, and anything else needed, then returns `cleaned_text` (str)

Reply:
That makes sense, do think it would be worth storing them in separate attributes? 

Also to clarify, where does removing the double spaces currently happen in `utils.py`?